components/generic/tabs/components/tabs.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { memo } from 'react'
+import cx from 'classnames'
+
+import { Tab } from '../types'
+
+export interface Props {
+  tabs: Tab[]
+  value: number | string
+  onChange: (event: React.ChangeEvent<{}>, value: number | string) => void
+  className?: string
+}
+
+const Tabs: React.FC<Props> = ({ tabs, value, onChange, className }) => {
+  const show = tabs.find((tab) => tab.id === value && tab.show !== false)
+
+  return (
+    <>
+      {show && (
```

Comment:
can you rename this show, so it's not the same variable name than the one in the map below?